spec/services/obo_token_authenticator_spec.rb

```diff
@@ -243,7 +243,7 @@ def generate_token(sub: 'rc://sk/8314451337_tisici_janko', exp: 1543437976, nbf:
     end
 
     it 'verifies EXP to IAT claim relation' do
-      token = generate_token(exp: (response.not_on_or_after + 41.minutes).to_i)
+      token = generate_token(exp: (response.not_on_or_after + 101.minutes).to_i)
```

Comment:
Preco tato magicka konstanta? Vytiahni to pls do nejakej `CONSTANT`

Reply:
Fixnute https://github.com/slovensko-digital/slovensko-sk-api/pull/43/commits/ac650c6d7165c32121391255b4fa12b2b3e5f9ee